decentraland/redux/reducers.js

```diff
@@ -95,6 +96,15 @@ function myAssets(state = { list: [] }, action) {
       const list = myAssets === null ? [] : myAssets;
       return { ...state, list };
     }
+    case SET_ACTION_ID_FOR_MY_ASSET: {
+      const { myAssetId: toUpdateId, actionId } = myAssetAndActionIds;
+      const { list: myAssets } = state;
+      const list = myAssets.map(asset => {
+        if (asset.id === toUpdateId) return { ...asset, actionId };
+        else return asset;
```

Comment:
An alternate approach could have been drilling down one level to just update the asset state itself rather than the entire list. This works fine too though! Just mentioning it for brainstorming purposes going forward.

Reply:
Resolving this